microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -190,15 +131,98 @@ You can access Collections from an InPrivate window when using a signed-out prof
 <!-- ====================================================================== -->
 ## Compose
 
-In the Copilot pane, the Compose tool uses AI to write for you, starting from a short prompt and **Tone**, **Format**, and **Length** selections. Text that's generated by Compose can be copied or inserted into the webpage directly.
+The Compose tool helps you draft text within the Microsoft Edge Sidebar, or rewrite selected text on any webpage.
+
+
+<!-- ------------------------------ -->
+#### Use Compose to draft text within the Edge Sidebar
+
+In the Copilot pane, the Compose tool uses AI to write for you, starting from a short prompt and **Tone**, **Format**, and **Length** selections:
 
 ![Compose UI](./index-images/compose.png)
 
-To access Compose, click the **Copilot** (![Copilot icon](./index-images/copilot-icon.png)) icon (**Ctrl+Shift+.**) in the upper right of Microsoft Edge, and then select **Compose**. Text and selections that you enter are sent to an internal Microsoft Large Language Model (LLM). 
+Text that's generated by Compose can be copied, or inserted directly into the webpage.
+
+To access the Compose tool, in the upper right of Microsoft Edge, click the **Copilot** (![Copilot icon](./index-images/copilot-icon.png)) button (**Ctrl+Shift+.**), and then select **Compose**.  Text that you select or enter is sent to an internal Microsoft Large Language Model (LLM). 
+
+To change settings for the sidebar or Copilot pane, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Sidebar** > **App and notification settings** > **Copilot**.
+
+
+<!-- ------------------------------ -->
+#### Use Compose to rewrite selected text on any webpage
+
+The Compose tool can help rephrase what you've written, and iterate using the same **Tone**, **Format**, and **Length** selections. 
+
+To access the Compose tool directly in a webpage, highlight text longer than five words in any input text field, and then click the **Rewrite with Copilot** button (**Alt+I**):
+
+![The Compose UI](./index-images/compose-inline.png)
```

Comment:
```suggestion
```
Not relevant to privacy

Reply:
removed colon, png line, & png.  Result:
```
To access the Compose tool directly in a webpage, highlight text longer than 
five words in any input text field, and then click the **Rewrite with Copilot** 
button.  Or, press **Alt+I**.

The Compose tool might not be available on some webpages or for certain input fields.
```